### PR TITLE
[doc] Add links to the latest presentations and add Twitter badge as well

### PR DESCRIPTION
**Pull request by vkarak**

### CODE REVIEW

**Comment by teojgo** [comment]
Change also the line where the setup.py starts: https://github.com/eth-cscs/reframe/blob/96ce404efe7afabb486fd9e4b5f2f1b79886e241/setup.py#L11

**Review by teojgo — CHANGES_REQUESTED** [review]
Only the setup.py needs to be updated otherwise it's fine.

File: setup.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from reframe import VERSION
 
 with open('README.md') as read_me:
-    long_description = ''.join(read_me.readlines()[16:])
+    long_description = ''.join(read_me.readlines()[18:])
```
Comment (teojgo) [inline setup.py:11]:
```suggestion
    long_description = ''.join(read_me.readlines()[17:])
```

**Review by teojgo — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (vkarak) [reply setup.py:11]: Why 17? The section starts in line 18 and 17 is a blank line. Is there a reason to have the first line blank?

**Review by teojgo — APPROVED** [review]

**Pull request merged**